### PR TITLE
Update docs with None parameters to use Optional

### PR DESCRIPTION
**Pull request by chrisnguyn**
Closes issue #1561.

### CODE REVIEW

**Comment by kx-chen** [comment]
@tiangolo We're ready for review.

File: docs_src/path_params_numeric_validations/tutorial001.py [hunk 1]
```diff
@@ -6,7 +8,7 @@
 @app.get("/items/{item_id}")
 async def read_items(
     item_id: int = Path(..., title="The ID of the item to get"),
-    q: str = Query(None, alias="item-query"),
+    q: Optional[str] = Query(None, alias="item-query"), # can someone check this?
```
Comment (Kludex) [inline docs_src/path_params_numeric_validations/tutorial001.py:11]: What should we check exactly?

File: docs_src/path_params_numeric_validations/tutorial002.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+from fastapi import FastAPI, Path, Query
 
 app = FastAPI()
 
 
 @app.get("/items/{item_id}")
 async def read_items(
-    q: str, item_id: int = Path(..., title="The ID of the item to get")
+    item_id: int = Path(..., title="The ID of the item to get"),
+    q: str = Query(None, alias="item-query"),
```
Comment (Kludex) [inline docs_src/path_params_numeric_validations/tutorial002.py:9]: I've added a suggestion, but I have the feeling this is related to the previous comment, am I right?
```suggestion
    q: Optional[str] = Query(None, alias="item-query"),
```

File: docs_src/query_params_str_validations/tutorial008.py [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+
 from fastapi import FastAPI, Query
 
 app = FastAPI()
 
 
 @app.get("/items/")
 async def read_items(
-    q: str = Query(
+    q: Optional[str] = Query( # can someone check if this is correct?
```
Comment (Kludex) [inline docs_src/query_params_str_validations/tutorial008.py:10]: .

**Comment by YKo20010** [comment]
@Kludex Hey Marcelo! I made the changes requested! Could you take another look at this issue?

**Comment by tiangolo** [comment]
Thanks for all the work here @chrisngyn , @YKo20010 , @kx-chen ! :nerd_face: :memo: :clap: :clap: :clap: 

And thanks for the help @Kludex ! :bow: :coffee: 

I updated a couple of things, a couple of tests and added a couple of extra notes.

Sadly, for some reason, I can't push those changes to your branch (as I normally would :thinking: ) so I created a new branch with all the changes here: https://github.com/tiangolo/fastapi/pull/1644 and I just merged it.

The good news is that all your commits (with authors) are preserved, you can now see your "contributor" badge in your comments on this same thread :smile: :tada:

I'll close this one now as it's all included there. :rocket: 

Thanks for your contribution! :cake: :tada:

**Pull request closed without merging**